src/line_drawer.py

```diff
@@ -231,20 +265,31 @@ def main(args):
         img = img.resize((basewidth, hsize), Image.ANTIALIAS)
 
     # grayscale using CIE luminance (Copied from http://linify.me)
-    img_arr = np.asarray(img, dtype=np.float)
+    img_arr = np.asarray(img, dtype=float)
     img_arr = 0.21 * img_arr[:, :, 0] + 0.72 * \
         img_arr[:, :, 1] + 0.07 * img_arr[:, :, 2]
     img_arr = img_arr.astype(np.int16)
 
     lines = compute_image_lines(
         img_arr, args.num_lines, args.num_lines_to_check, draw_type, args.line_heaviness)
-    output_image_arr = draw_line_image(
-        lines, img_arr.shape, draw_type, args.line_heaviness)
+    
+    if str.upper(args.output_format) == 'SVG':
+        stroke_width = args.line_heaviness * 0.01
```

Comment:
I c didn't understand the line_heaviness and its purpose so I reused it initially.

Reply:
Currently line_heaviness should describe how much intensity is added or subtracted for the pixel under the line. E.g. for a subtractive image where the background is first white, i.e. intensity = 255, for each line the pixel intensity of the line_heaviness is subtracted. This was needed as drawing a fully black line would quickly lead to a black images where no nuances are visible anymore.

If you have a better name for that, just shoot.

Reply:
IMHO, it would make more sense to call it a line weight or line weight bias.

As you may have noticed, I didn't implement the variable in the SVG output code.
There are three options:
1. Setting [opacity attribute](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/opacity) (float from 0.0 to 1.0) for each line;
2. Setting a stroke colour with alpha channel (RGBA) to each line;
3. Varying stroke width for each line;

The first one seems the most rational to me as that would produce shorter SVG code.

Reply:
Perfect, line weight is what I was looking for but did not found. Therefore, could you change `line_heaviness` to `line_weight` in the code?

I agree too to use the opacity attribute as it corresponds also the most to the line weight from my understanding.

Although the results between the SVG  and PNG mode do not look exactly the same I would suggest to use as standard values the following: `stroke-width=1.0` and `stroke-opacity=0.04`. 

With this the stroke width would be 1 pixel and the opacity would correspond to the line weight with 10/256≈0.04.